src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -2678,6 +2689,32 @@ public void testTenantAwareDeleteSamlProviderConfig() throws Exception {
     checkUrl(interceptor, "DELETE", expectedUrl);
   }
 
+  @Test
+  public void testCreateOidcProviderFromEmulatorAuth() throws Exception {
```

Comment:
Optional: Add another test for any of the user management flows, like `GetUser`.